Exercise_4/css/partials/side-menu/_small-screen.scss

```diff
@@ -0,0 +1,82 @@
+@import '../variables';
+
+/*SIDE-MENU*/
+
+@media only screen and (max-width: $small-screen_max-width) {
+
+  .side-menu {
+    cursor: pointer;
+    display: flex;
+    font-size: 2rem;
+    margin: 0 2rem;
+    width: 10vw;
+
+    &__content-wrapper {
+      display: none;
+      padding: 1rem;
+    }
+
+    &:focus {
```

Comment:
Unfortunately, nesting here looks bad. Make it in one row: `&:focus *something here* {`